prow/plugins/projectmanager/projectmanager.go

```diff
@@ -0,0 +1,208 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package projectmanager is a plugin to auto add pull requests to project boards based on specified conditions
+package projectmanager
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/sirupsen/logrus"
+
+	"k8s.io/test-infra/prow/github"
+	"k8s.io/test-infra/prow/pluginhelp"
+	"k8s.io/test-infra/prow/plugins"
+)
+
+const (
+	pluginName = "project-manager"
+)
+
+// TODO Create a new handler for issues, look in hook/server.go
+// TODO Handle Label deletion, pr/issue should be removed from the project when label criteria does  not meet
+// TODO Pr/issue state change, pr/iisue is on project board only if its state is listed in the configuration
+func init() {
+	plugins.RegisterPullRequestHandler(pluginName, handlePullRequest, helpProvider)
+}
+
+func helpProvider(config *plugins.Configuration, enabledRepos []string) (*pluginhelp.PluginHelp, error) {
+	pluginHelp := &pluginhelp.PluginHelp{
+		Description: "The project-manager plugin automatically adds Pull Requests to specified GitHub Project Columns, if the label on the PR matches with configured project and the column.",
+		Config: map[string]string{
```

Comment:
This isn't how `Config` is meant to be used. Help providers are functions that take the current config and the list of enabled repos so that they can return dynamically generated help information specific to the current configuration.
`Config` needs to describe the current configuration, not provide a hard coded configuration example.
Here is an example from the `approve` plugin: https://github.com/kubernetes/test-infra/blob/be2fa898afec153f9d46f17c95a3f595d97dc29e/prow/plugins/approve/approve.go#L114-L134

Reply:
I am a bit confused and not sure how to test this locally, but I changed code to create a readable structure from the 'Config' provided.